test/graphql/types/Mutation/createChatMembership.test.ts

```
@@ -0,0 +1,604 @@
+				try {
+					// Your code that might throw an error
+				  } catch (error: unknown) {
+					if (error instanceof Error) {
+					  // Now TypeScript knows that 'error' is an instance of Error
+					  console.error(error.message); // Access 'message' property
+					  expect(error).toBeInstanceOf(TalawaGraphQLError);
+					} else {
+					  // Handle other types of errors, if needed
+					  console.error("An unknown error occurred");
+					}
+				  }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)